docs/input/docs/upgrading/1.x-to-2.x.md

```diff
@@ -0,0 +1,211 @@
+---
+Order: 10
+Title: Upgrading from 1.x to 2.x
+Description: Changes and suggestions needed for porting from Cake.Recipe 1.x to 2.x
+---
+
+In the 2.0.0 release of Cake.Recipe, there were a total of 15 breaking changes.  As a result, there are a number of changes that have to be made to any project that is currently using a 1.x.x release of Cake.Recipe, in order to allow it to start using this release.
+
+Have a look at this [pull request](https://github.com/cake-contrib/Cake.DotNetVersionDetector/pull/59) on the Cake.DotNetVersionDetector repository for an example of a project that has made this transition.
+
+## Required Changes
+
+### Stop using username and password combination for accessing GitHub
+
+As of November 2020, GitHub is no longer going to support using a combination of username/password for accessing its API.  Instead, you _have_ to use a Personal Access Token.  As a result, the ability to provide a username/password via environment variables into Cake.Recipe has been removed.  Going forward, you will have to provide an environment variable for the [GITHUB_TOKEN](../fundamentals/environment-variables#github_token).
```

Comment:
As in `index.cshtml`: Did you mean "\_have\_ to" or rather "_have to_"?

Reply:
:+1: